ginga/misc/plugins/ChangeHistory.py

```diff
@@ -256,3 +256,10 @@ def start(self):
 
     def __str__(self):
         return 'changehistory'
+
+
+# Replace module docstring with config doc for auto insert by Sphinx.
+# In the future, if we need the real docstring, we can append instead of
+# overwrite.
+from ginga.util.toolbox import generate_cfg_example
+__doc__ = generate_cfg_example('plugin_ChangeHistory', package='ginga')
```

Comment:
Yes. For each plugin, we will need to add an incantation to its RST file, and this to the actual code (both of which are applied to `ChangeHistory` as part of this PR to illustrate its usage). Note that I am completely overwriting the plugin's module docstring here. Currently this is not an issue because the documentation does not use it, so instead I use it to auto-generate the config file.

Another alternative is to actually document the entire plugin completely within the module docstring (i.e., the intro, usage, screenshot, etc.). In that case, we *append* the cfg contents instead using `__doc__ += ...`. However, I didn't do it because it requires more work porting existing plugin documentation back to the code.